x-pack/plugins/apm/server/lib/search_strategies/correlations/get_query_with_params.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { estypes } from '@elastic/elasticsearch';
+import {
+  TRANSACTION_DURATION,
+  TRANSACTION_NAME,
+  PROCESSOR_EVENT,
+  SERVICE_NAME,
+} from '../../../../common/elasticsearch_fieldnames';
+import type { SearchServiceParams } from '../../../../common/search_strategies/correlations/types';
+import { environmentQuery as getEnvironmentQuery } from '../../../utils/queries';
+
+export enum ProcessorEvent {
+  transaction = 'transaction',
+  error = 'error',
+  metric = 'metric',
+  span = 'span',
+  profile = 'profile',
+}
+
+const getRangeQuery = (
```

Comment:
Does this have different behavior than [`rangeQuery`](https://github.com/elastic/kibana/blob/1e7ef987bebc611fe4b8bc7bd8da194e924f75a0/x-pack/plugins/observability/server/utils/queries.ts#L21-L37)?

Reply:
Updated to using observability's rangeQuery here https://github.com/elastic/kibana/pull/99905/commits/4686225257ec6ed458a15595911da8838a6c15d9